Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -62,7 +56,27 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     @Override
     public void onResume() {
         super.onResume();
-        payPalLauncher.handleReturnToAppFromBrowser(requireContext(), requireActivity().getIntent());
+        PayPalPendingRequest.Started pendingRequest = getPendingRequest();
+        if (pendingRequest != null) {
+            PayPalPaymentAuthResult paymentAuthResult = payPalLauncher.handleReturnToAppFromBrowser(pendingRequest, requireActivity().getIntent());
```

Comment:
```suggestion
            payPalClient.continueTokenize(pendingRequest, (result) -> { /* handle result logic */ };
```
nit: We could have a "continue" / "finish" method to pass control back to the SDK to complete the flow.

Reply:
Yeah it's take or leave. This is another case where PPCP may be different. We probably won't have a Launcher since we only have browser-based flows at the moment.

Reply:
I've added this to the ticket to discuss the `PendingRequest` object naming - I think we can determine a good method naming pattern that makes sense with whatever we decide to call that request object as well.